src/Components/Publishing/Byline/__tests__/__snapshots__/NewsByline.test.tsx.snap

```diff
@@ -158,7 +158,7 @@ exports[`News Byline renders on mobile 1`] = `
   >
     <a
       className="c6"
-      href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.artsy.net%2Farticle%2Fundefined"
+      href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.artsy.net%2Farticle%2Fnews-article"
```

Comment:
Is it intentional that all of these snapshots got updated?

Reply:
No, i haven't even run tests- all i've done is rebase off master. i'll double check this.